app/components/admin/participants/table_row.rb

```diff
@@ -13,6 +13,12 @@ def initialize(profile:)
 
       attr_reader :profile
       delegate :school, to: :profile
+
+      def validation_status
+        return { text: "Not ready", colour: "grey" } unless profile.npq?
+
+        { text: "Pending", colour: "yellow" }
+      end
```

Comment:
Is this used?

Reply:
Ah, no. It was replaced by the extracted component! :)